versions/draft-3/SPEC.md

```diff
@@ -705,6 +721,11 @@ task t {
   - Two input files which originated in the same storage directory must also be localized into the same directory for task execution (see the special case handling for Versioning Filesystems below).
 - When a WDL author uses a `File` input in their [Command Section](#command-section), the fully qualified, localized path to the file is substituted into the command string.
 
+##### FileURI localization
+`FileURI` inputs also follow specific localization rules:
+- If the URI in the FileURI is a file mounted on the filesystem (e.g. `file:///home/foo/test.txt` or `/home/foo/test.txt`), then the File referenced at must be made accessible to the job at execution time by the engine.
+- If the URI points to an object storage URI (e.g. `gs://my-bucket/test.txt` for Google Cloud Storage) then FileURIs are essentially treated as Strings as far as localization.
```

Comment:
I wouldn't even call out `String` here (`String` localization is "do nothing", but we might as well say that directly). eg:

`"If the URI [...] Cloud Storage) then an existence and accessibility check should be made but no further action is required for localization.`

Reply:
What about if it's `http://`?

Reply:
Not addressing `http://` for now.  Done, otherwise.

@cjllanwarne How would you want to handle http?  I would limit FileURI to local filesystems and cloud storage.  However, that can be confusing for end users, so I am open to suggestions.  Or maybe have `http://` be handled by `File`?